CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ## Updated
 * Updated the vendored version of DataDog/datadog-go which fixes parsing for abstract unix domain sockets in the statsd client. Thanks, [androohan](https://github.com/androohan)!
+* Veneur uses sentry-go instead of the deprecated Sentry package, raven-go. Thanks, [yanke](https://github.com/yanske1)!
```

Comment:
nit: we usually phrase these in the imperative (ie: "Switched to" or "Migrated to")